.travis.yml

```diff
@@ -23,11 +23,21 @@ env:
         # Add the TRAVIS_TOKEN environment variable for the "conda-forge" Travis user.
         - secure: "BBJrw5OcEG4zEUxjXt2txNoz+o7ilTb0e8Tj1626USKqi6HAWOdSFHW7PLYR6vyefWsHnGLAeYMRr9UB56O2oDwS7jgClBKbyut2+Q+8SQBwEtNCMHdU996IzSuDP4AM3h7DLjTsuinWhwpqLu72z2ZjE9OMz4q03SyyiBOecWU1btYbp/u3bpNzyhb2u0HMkRUnS8MxPUa8aHZBKp/JpLrGQmIT0LcjLNp4/jTZp+cs+J//02hAhJHV3vMmyefVvRZayu3xKzutpb3qIga0JAP3d0j2FWXRHVTW6Ke5H4u/4NU7iTpdRjpymnzH7H2kfOrNzgCJ23NfNhScDJDc5URvdXlstgkC4lDkd7BbSVgKKaVy2vVcMkz98qbBnhi0akMHNZXPLvT9tec9AsAX5/pdNaa1rH/iRhjWRfTCnqODX+RTwMhA1/3rkmhzI/8JFCcaZet8+lA2LtOf95jGmwcRwPlNFX0Jdd1QBk2nUAUpCePWIdg7GRxYuO7a4QCbhc6bSyB/WrCQZ1oEmtwGSRqxpMKvQmjxJObg40MgP0DaquGov30A4QLoEdvXCnM7w3QyIr5L/BrLlnCy8LesAZIBn2l5LAHrBFGh9t/lCK9APlUjD9dgT6eCYOSbfdBghKCGsajAsM/fwsPvcjOmpFCLf+UZTer36B57ccHrYdU="
 
+    #matrix:
+    #    - TEST_SPLIT=0/5
+    #    - TEST_SPLIT=1/5
+    #    - TEST_SPLIT=2/5
+    #    - TEST_SPLIT=3/5
+    #    - TEST_SPLIT=4/5
```

Comment:
Is this suppose to be commented?

Reply:
yes, when we want to run this script it has to be uncommented

Reply:
Would it make sense to leave this uncommented so we can easily run it? That or should we add a comment above explaining this should be uncommented to run the script?

Reply:
Leaving this uncommented would add 5 builds to each PR run here.

Reply:
Sorry missed this was going in `master` for some reason even though it is clearly in the title. :man_facepalming:

Reply:
I guess just a comment above should suffice.